components/image_viewer/image_viewer.vue

```diff
@@ -120,6 +120,14 @@ export default {
       type: String,
       required: true,
     },
+
+    /**
+     * Forces the image to stay open
+     */
+    forceOpen: {
+      type: Boolean,
+      default: null,
+    },
```

Comment:
If we're going to do this.. we might as well just put in open.sync (or show.sync?) functionality. This will not only help for these visual tests but also for any developer that needs to trigger this in a different way.

see `dropdown.vue` for an example of this. It would have default behaviour if open is null (open on click), but if set by the consumer it would disable the default open behaviour and allow the consumer to control it instead.

https://v2.vuejs.org/v2/guide/components-custom-events.html#sync-Modifier

The equivalent in vue 3 is `v-model:open `